### PR TITLE
Updated to final PSR-15

### PR DESCRIPTION
**Pull request by geerteltink**

### CODE REVIEW

File: src/AuthenticationMiddleware.php [hunk 1]
```diff
@@ -1,18 +1,18 @@
 <?php
 /**
  * @see       https://github.com/zendframework/zend-expressive-authentication for the canonical source repository
- * @copyright Copyright (c) 2017 Zend Technologies USA Inc. (https://www.zend.com)
+ * @copyright Copyright (c) 2018 Zend Technologies USA Inc. (https://www.zend.com)
```
Comment (michalbundyra) [inline src/AuthenticationMiddleware.php]: please use range instead

File: test/AuthenticationMiddlewareTest.php [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
  */
 
 declare(strict_types=1);
 
 namespace ZendTest\Expressive\Authentication;
 
-use Interop\Http\Server\MiddlewareInterface;
-use Interop\Http\Server\RequestHandlerInterface;
+use Psr\Http\Server\MiddlewareInterface;
+use Psr\Http\Server\RequestHandlerInterface;
```
Comment (michalbundyra) [inline test/AuthenticationMiddlewareTest.php]: Please use alphabetical order.

File: test/AuthenticationMiddlewareTest.php [hunk 3]
```diff
@@ -1,16 +1,16 @@
 <?php
 /**
  * @see       https://github.com/zendframework/zend-expressive-authentication for the canonical source repository
- * @copyright Copyright (c) 2017 Zend Technologies USA Inc. (https://www.zend.com)
+ * @copyright Copyright (c) 2018 Zend Technologies USA Inc. (https://www.zend.com)
```
Comment (michalbundyra) [inline test/AuthenticationMiddlewareTest.php]: please use range instead

File: composer.json [hunk 4]
```diff
@@ -21,12 +21,14 @@
     },
     "require": {
         "php": "^7.1",
-        "http-interop/http-server-middleware": "^1.0.1",
         "psr/container": "^1.0",
-        "psr/http-message": "^1.0.1"
+        "psr/http-message": "^1.0.1",
+        "psr/http-server-handler": "^1.0",
+        "psr/http-server-middleware": "^1.0",
+        "zendframework/zend-diactoros": "^1.7"
```
Comment (michalbundyra) [inline composer.json]: Why new dependency to `zendframework/zend-diactoros` here?

File: composer.json [hunk 5]
```diff
@@ -21,12 +21,14 @@
     },
     "require": {
         "php": "^7.1",
-        "http-interop/http-server-middleware": "^1.0.1",
         "psr/container": "^1.0",
-        "psr/http-message": "^1.0.1"
+        "psr/http-message": "^1.0.1",
+        "psr/http-server-handler": "^1.0",
```
Comment (michalbundyra) [inline composer.json]: We don't need it, as it is required by `psr/http-server-middleware`

File: composer.json (hunk 4 above)
Comment (geerteltink) [reply composer.json]: https://github.com/zendframework/zend-expressive-authentication/blob/release-1.0.0/src/ResponsePrototypeTrait.php#L14

Comment (michalbundyra) [reply composer.json]: The whole point is not to have it there !

https://github.com/zendframework/zend-expressive-authentication/blob/release-1.0.0/src/ResponsePrototypeTrait.php#L26-L33

Comment (geerteltink) [reply composer.json]: https://github.com/zendframework/zend-expressive-authentication/blob/release-1.0.0/src/ResponsePrototypeTrait.php#L37

**Review by michalbundyra — APPROVED** [review]

**Comment by weierophinney** [comment]
Thanks, @xtreamwayz!

**Pull request merged**